src/main/java/com/org/jenkins/custom/jenkins/distribution/service/services/UpdateCenterService.java

```diff
@@ -37,14 +36,15 @@ public JSONObject downloadUpdateCenterJSON() throws Exception {
             File updateCenterFile = File.createTempFile("update-center", ".json");
             updateCenterFilePath = updateCenterFile.getPath();
             LOGGER.info("Creating a new file" + updateCenterFile.getPath());
-            HttpGet get = new HttpGet(UPDATE_CENTER_JSON_URL);
             LOGGER.info("Executing Request at " + UPDATE_CENTER_JSON_URL);
-            CloseableHttpClient httpClient = HttpClients.createDefault();
-            CloseableHttpResponse response = httpClient.execute(get);
-            responseString = EntityUtils.toString(response.getEntity());
-            byte[] buf = responseString.getBytes(StandardCharsets.UTF_8);
+            try (CloseableHttpClient httpClient = HttpClients.createDefault();
+                 CloseableHttpResponse response = httpClient.execute(new HttpGet(UPDATE_CENTER_JSON_URL))) {
+                responseString = EntityUtils.toString(response.getEntity());}
```

Comment:
Please fix the indentation here, it will clarify the intent of the try-with-resources statement.

Reply:
Thanks Martin working on them